java/core/src/java/org/apache/orc/OrcConf.java

```diff
@@ -190,6 +193,14 @@ public enum OrcConf {
                       + "determined, they are combined using AND. The order of application is "
                       + "non-deterministic and the filter functionality should not depend on the "
                       + "order of application."),
+
+  PLUGIN_FILTER_ALLOWLIST("orc.filter.plugin.allowlist",
+                          "orc.filter.plugin.allowlist",
+                          "*",
+                          "A comma-separated list of allowlist classes. It is used to specify the "
```

Comment:
It looks like recursive definition. We cannot use `allowlist` in the definition sentence of `allowlist`. Maybe, the following is better?
```
- A comma-separated list of allowlist classes
+ A list of comma-separated class names
```